src/config.rs

```diff
@@ -428,6 +435,17 @@ macro_rules! build_cf_opt {
         if $opt.enable_doubly_skiplist {
             cf_opts.set_doubly_skiplist();
         }
+        if let Some(accessor) = $region_info_accessor {
```

Comment:
raise an error if enable_compaction_guard && accessor is None ?

Reply:
hmm, I made it optional to get around some of the test failures, where the region_info_accessor is not available. How about just print a warning?

Reply:
good idea